aiida-backup.sh

```diff
@@ -4,14 +4,19 @@
 
 show_help() {
     echo
-    echo "Usage: ./aiida-backup.sh [-h] [-e env-type] [-c conda-env] [-v venv-path] [-p projects-path] [-n project-name]"
+    echo "Usage: ./aiida-backup.sh [-h] [-e env-type] [-c conda-env] [-v venv-path] [-n project] [-p profiles]"
+    echo
+    echo "Examples:"
+    echo "  conda:            ./aiida-backup.sh -e conda -c <conda-env> -n <project>"
+    echo "  venv:             ./aiida-backup.sh -e venv -v aiida2.5 -n <project>"
+    echo "  aiida-project:    ./aiida-backup.sh -e aiida-project -n <project>"
     echo
     echo "Options:"
     echo "  -h                Show this help message"
     echo "  -e env-type       The type of environment to activate (conda, venv, aiida-project)"
     echo "  -c conda-env      The name of the Conda environment to activate"
     echo "  -v venv-path      The path to the virtual environment to activate"
-    echo "  -n project        The name of the AiiDA project"
+    echo "  -n project        The name of the AiiDA project. This is not related to aiida-project but a personal choice for your project."
```

Comment:
We shortly talked about this that this is actually the aiida project name in case aiida-project is used as enviroment right? So it should be 
```suggestion
    echo "  -n project        The name of you want to give your project using the environment and profiles. In the case of aiida-project as env-type, this is the name of the aiida-project."
```

Reply:
Too much 😅 Also, I can't remember now, but didn't we say something like making this optional and using a "default" project if none provided?